clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -45,6 +46,8 @@ func createSimpleTestExecutor() TestExecutor {
 
 // ExecuteTest executes test based on provided configuration.
 func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) *util.ErrorList {
+	framework.AutomanagedNamespaceName = fmt.Sprintf("namespace-%v", rand.Int())
```

Comment:
Why do we need randomization?
Why can't we do sth like "test-<test_name from config>" or sth like that?

Names should generally be different anyway :)

Reply:
> sth like "test-<test_name from config>" 

I thought about this. In most cases it is a proper solution.
The only case it doesn't solve is when somebody runs the same twice. This is allowed, so IMHO this also be handled correctly.

Reply:
OK - that's a bit artificial, but I agree that probably worth supporting.

Then let's do: "test-<test name>-<hash>" (but let make hash smaller than int32 and with constant amount of characters)

Reply:
Done. Turns out this hash needs to be in [a-z0-9].